app/api.py

```diff
@@ -27,6 +27,11 @@
 import StringIO
 import xml.dom.minidom
 
+#For GA MP
```

Comment:
Can you write "Google Analytics Measurement Protocol" instead of "GA MP"? Avoid abbreviations/acronyms in general, unless it's well known and commonly used in the project. Same for below.

Also, put a whitespace between "#" and the comment. Person Finder follows this style guide:
https://google.github.io/styleguide/pyguide.html
(Note that this is slightly different from Google's internal Python style guide.)